src/main/java/io/vertx/kafka/client/consumer/KafkaConsumer.java

```diff
@@ -597,4 +597,16 @@ default void close() {
    */
   @GenIgnore
   Consumer<K, V> unwrap();
+
+  /**
+   * Sets the polling timeout (in ms) for the underlying native Kafka Consumer. Defaults to 1000.
+   * Setting timeout to a lower value results in a more 'responsive' client, because it will block for a shorter period
+   * if no data is available in the assigned partition and therefore allows subsequent actions to be executed with a shorter
+   * delay. At the same time, the client will poll more frequently and thus will potentially create a higher load on the Kafka Broker.
+   *
+   * @param timeout The time, in milliseconds, spent waiting in poll if data is not available in the buffer.
+   * If 0, returns immediately with any records that are available currently in the native Kafka consumer's buffer,
+   * else returns empty. Must not be negative.
+   */
+  KafkaConsumer<K, V> pollingTimeout(long timeout);
```

Comment:
Just a minor comment. What do you think about using `pollTimeout` instead of `pollingTimeout` ? Of course this change should be reflected in all the other parts where `polling` is used (i.e. `KafkaReadStream`, ...)